nixos/modules/services/networking/haproxy.nix

```diff
@@ -26,6 +26,13 @@ with lib;
         '';
       };
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.haproxy;
+        defaultText = literalExpression "pkgs.haproxy";
+        description = lib.mdDoc "HAProxy package to use.";
+      };
+
```

Comment:
Use `mkPackageOptionMD` like in https://github.com/NixOS/nixpkgs/pull/237824/files#diff-54db11ee9dbd51466b032f18228d1ddc8da76ada6fb2306ccffb7e7b8cfc4f8dR14